packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -190,6 +191,40 @@ export class Meter implements api.Meter {
     return valueObserver;
   }
 
+  /**
+   * Creates a new `UpDownSumObserver` metric.
+   * @param name the name of the metric.
+   * @param [options] the metric options.
+   * @param [callback] the value observer callback
+   */
+  createUpDownSumObserver(
+    name: string,
+    options: api.MetricOptions = {},
+    callback?: (observerResult: api.ObserverResult) => unknown
+  ): api.ValueObserver {
```

Comment:
`api.ValueObserver` -> `api.UpDownSumObserver` ?

Reply:
Since all observers have the same interface, shouldn't it just be `api.Observer`?

Reply:
Since it's just a type alias, that would be equivalent right now. I think the idea is to be more future proof in case the types diverge.

Reply:
they have for now but we should use the final one anyway